google-cloud-bigquery/clirr-ignored-differences.xml

```diff
@@ -129,4 +129,9 @@
     <className>com/google/cloud/bigquery/StandardTableDefinition*</className>
     <method>*BigLakeConfiguration(*)</method>
   </difference>
-  </differences>
\ No newline at end of file
+  <difference>
+    <differenceType>7005</differenceType>
```

Comment:
```suggestion
    <differenceType>7013</differenceType>
```

It looks like it's now complaining about a new `abstract` method, I assume on the theory that subclasses would have to implement it. Given that the only subclass here will be the one generated by AutoValue (I hope :)), we should be able to suppress that, too.

Since the old method continues to exist in this iteration of the PR, that suppression can replace the old one from the time of the method removal. With that change, `JOB_TYPE=clirr .kokoro/build.sh` appears to pass.